packages/drivers/driver-web-cache/src/FluidCacheIndexedDb.ts

```diff
@@ -73,8 +73,10 @@ export function getFluidCacheIndexedDbInstance(
 
 // Deletes the indexed DB instance.
 // Warning this can throw an error in Firefox incognito, where accessing storage is prohibited.
-export function deleteFluidCacheIndexDbInstance(): Promise<void> {
-	return deleteDB(FluidDriverCacheDBName);
+export function deleteFluidCacheIndexDbInstance(
+	deleteDBCallbacks?: DeleteDBCallbacks,
```

Comment:
Should we have jsdocs here to document what this parameter would be used for?

Reply:
I could add them but I didn't see them anywhere in these functions

Reply:
I find this comment in the idb docs interesting as well: 

"Called if the database already exists and there are open connections that don’t close in response to a versionchange event, the request will be blocked until they all close."

Wonder if our version change handler is missing some code to auto close (not in scope for this PR, just a thought)

Reply:
Oh I didn't know there was an existing versionchange handler, let me check

Reply:
I don't see any existing `versionchange` handlers, are there any? This is more of a hotfix PR to close DBs and (more importantly) provide a delete DB callback mechanism to detect hanging conditions when they will occur. However, I agree that there is a potential for `versionchange` listeners to act more well-behaved but think it should be a thoughtfully designed pattern backed up by unit tests and will take a bit more time.

Reply:
I was thinking of this as a follow up, not anything for this PR. So we are on the same page